plugins/obs-ffmpeg/obs-ffmpeg-output.c

```diff
@@ -68,14 +70,16 @@ struct ffmpeg_data {
 
 	uint64_t           start_timestamp;
 
-	int64_t            total_samples;
+	int64_t            total_samples[MAX_AUDIO_MIXES];
 	uint32_t           audio_samplerate;
 	enum audio_format  audio_format;
 	size_t             audio_planes;
 	size_t             audio_size;
-	struct circlebuf   excess_frames[MAX_AV_PLANES];
-	uint8_t            *samples[MAX_AV_PLANES];
-	AVFrame            *aframe;
+	int                num_audio_streams;
+	int                audio_tracks;//bitmask storing the indexes of the mixes selected
```

Comment:
Style:  This line is too long.  Consider moving the comment to its own line, or shortening the comment.